classes/image_types_ostree.bbclass

```diff
@@ -198,11 +198,7 @@ IMAGE_CMD_garagesign () {
             garage-sign init --repo ${GARAGE_SIGN_REPO} --home-dir ${GARAGE_SIGN_REPO} --credentials ${SOTA_PACKED_CREDENTIALS}
         fi
 
-        if [ -n "${GARAGE_SIGN_REPOSERVER}" ]; then
-            reposerver_args="--reposerver ${GARAGE_SIGN_REPOSERVER}"
-        else
-            reposerver_args=""
-        fi
+        reposerver_args="--reposerver $( unzip -p ${SOTA_PACKED_CREDENTIALS} tufrepo.url )"
```

Comment:
Do you need to cat that URL or can we just use the file as is?

Reply:
That is the interface as was described by Simao. We can ask @simao or @jochenschneider to process tufrepo.url internally instead, but it doesn't work yet to my understanding.

Reply:
No problem, just checking.